src/Features/Core/Portable/UseAutoProperty/AbstractUseAutoPropertyCodeFixProvider.cs

```diff
@@ -55,10 +55,10 @@ public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
                     ? CodeActionPriority.Low
                     : CodeActionPriority.Medium;
 
-                context.RegisterCodeFix(
-                    new UseAutoPropertyCodeAction(
+                context.RegisterCodeFix(CodeAction.SolutionChangeAction.Create(
                         AnalyzersResources.Use_auto_property,
                         c => ProcessResultAsync(context, diagnostic, c),
+                        equivalenceKey: null,
```

Comment:
Is this intentional? I think `nameof(AnalyzersResources.Use_auto_property)` would match equivalence key used in other IDE fixers.

Reply:
we didn't have an equikey before.  but i'm fine adding one here.

Reply:
done.